File: java/GroceryList.java
import java.util.Scanner;
import java.util.Arrays;

public class GroceryList{
    public static void main(String...args){
       
        int index = 0;
        Scanner scanner = new Scanner(System.in);
        System.out.println("""
                        This is your grocery cart
                        where you keep track of all the packages that you 
                        List the grocery items you want to add to your cart
                        press:
                        1. To add item
                        2. To remove item
                        3. To show availaible items
                         0.to exit
                         """);

      
        String Choice = scanner.nextLine();
        int newChoice = Integer.parseInt(Choice);

        System.out.println("Enter the number of item you may want to add");
        int noItems = scanner.nextInt();
        String[] list = new String [noItems];


switch(newChoice){

     case 1:{System.out.println("Enter the item you may want to add");
             while (index <= noItems){
                String item = scanner.nextLine();
                list[index] = item;
                noItems++;
              System.out.println(Arrays.toString(list));
             }


    }

    case 3: System.out.println(list);
             


    


}
     

}
}

File: python/grocery_list.py



print("This is your grocery cart \
where you keep track of all the packages that you need")

users_list = int(input("""List the grocery items you want to add to your cart\
press:
1. To add item and display entire grocery list
2. To remove item
 0.to exit
 """))

users_cart = []




match users_list:
        case 1:
#            new =  int(input("2 to braek"))
#            while(new != 2): 
            index = 0 
            number_of_items = int(input("Enter the number of items you may want to add to your cart:"))
            while (index <= number_of_items):
                item = input("Enter the items you may want to add to your cart: ")
                users_cart.append(item)
                index += 1
            print("\n") 
            for items in users_cart:
                print(f"your items are: {item}")
            print(f"your grocery list is:\n {users_cart}")

        case 2:
            count = 0
            number_of_removal = int(input("how many items do you want to remove: "))
            while(count <= number_of_removal):
                item_to_remove = input("Enter the one item you may want to remove to your cart: ")
                users_cart.remove(users_cart[ item_to_remove])
           
          
          
          
           
            


